client-v2/src/constants/polling.ts

```diff
@@ -0,0 +1,3 @@
+export const collectionsPollInterval = 10000;
+export const articlesPollInterval = 30000;
+export const ophanPollInterval = 300000;
```

Comment:
Could we do an initial poll for data once collections have loaded, and then default to this longer polling interval?

Reply:
Thanks for noticing, this is really important to address 👍